src/twisted/words/xish/domish.py

```diff
@@ -807,11 +807,18 @@ def _onStartElement(self, name, attrs):
             qname = ('', name)
 
         # Process attributes
+        new_attrs = {}
```

Comment:
Twisted's coding standard is slightly different than PEP8; this would properly be `newAttrs` and `toDelete`.
```suggestion
        newAttrs = {}
```